src/Component/Filter/FilterTree/FilterTree.tsx

```diff
@@ -0,0 +1,567 @@
+import * as React from 'react';
+
+import {
+  Tree,
+  Icon,
+  Dropdown,
+  Menu,
+  Button,
+  Tooltip
+} from 'antd';
+
+const _get = require('lodash/get');
+const _set = require('lodash/set');
+
+const TreeNode = Tree.TreeNode;
+
+import {
+  Filter as GsFilter,
+  ComparisonFilter as GsComparisonFilter
+} from 'geostyler-style';
+
+import './FilterTree.css';
+
+import {
+  Data as Data
+} from 'geostyler-data';
+
+import ComparisonFilterUi from '../ComparisonFilter/ComparisonFilter';
+import { TreeNode } from 'antd/lib/tree-select';
```

Comment:
You already defined TreeNode in line 15. Looks like you are running into naming conflicts.